internal/web3ext/web3ext.go

```diff
@@ -733,6 +733,12 @@ web3._extend({
 			params: 1,
 			inputFormatter: [null]
 		}),
+		new web3._extend.Method({
+			name: 'getValidatorsPublicKeys',
```

Comment:
nit:  Perhaps rename to `getValidatorBLSPublicSigningKeys`, to differentiate from the `ECDSA consensus signing keys`.

Reply:
Good idea.